backend/schema/metadata.ts

```diff
@@ -18,7 +19,7 @@ export const metadataProperties = {
   modelNumber: {
     type: "string",
   },
-  modelType: {
+  modelVairant: {
```

Comment:
Typo – it is Variant, not Vairant.

But I think we can keep the name as modelType – why the change? The original schema had modelType as residential/commercial vs industrial, but I think we can call that a different name. I think modelType is a good name for the distinction between heat pump, dryers, water heaters, etc.

But let me know what you are thinking. We can also merge this now to unblock Ricky and then change the name later.